src/app/shared/services/confirmation.service.spec.ts

```diff
@@ -46,6 +46,28 @@ describe('ConfirmationService', () => {
     });
   });
 
+  it('should open the dialog and allow for HTML tags but not render any script embedded in the message', () => {
```

Comment:
I think this should not be tested here since it's the responsibility of the ConfirmDialogComponent.